R/get_PITobs.R

```diff
@@ -46,11 +46,11 @@ get_PITobs = function(query_type = c('obs_site', 'release_site'),
   rear_type <- match.arg(rear_type, several.ok = FALSE)
 
   # set up default start and end days
-  start_day <- substr(start_date,1,5)
-  syear <- substr(start_date, 7,10)
+  start_day <- gsub('-','/',substr(as.Date(start_date, format ='%m/%d/%Y'),6,10))
```

Comment:
If we figure out a way to ensure the string format is followed for the date values I think this is more complex than it needs to be.